Python/pylifecycle.c

```diff
@@ -651,6 +651,8 @@ pycore_create_interpreter(_PyRuntimeState *runtime,
     }
     assert(_Py_IsMainInterpreter(interp));
 
+    interp->lazy_imports_enabled = 0;
```

Comment:
I suggest doing this init inside `PyInterpreterState_New` so you get it initialized for every new interpreter state (this isn't the only place where a new interpreter state is created)

Reply:
I see some `int` values is initialized in `init_interpreter`, a function called in `PyInterpreterState_New`. Finally, found the initialization place in the code sea. 😂